readthedocs/api/v3/views.py

```diff
@@ -161,6 +178,33 @@ def get_view_description(self, *args, **kwargs):  # pylint: disable=arguments-di
                 return mark_safe(description.format(project_slug=project.slug))
         return description
 
+    def create(self, request, *args, **kwargs):
+        """
+        Override method to importing a Project.
+
+        * Save the Project object
+        * Assign the user from the request as owner
+        * Sent project_import signal
+        * Trigger an initial Build
+        """
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        project = serializer.save()
+        headers = self.get_success_headers(serializer.data)
+
+        # TODO: these lines need to be adapted for Corporate
+        project.users.add(request.user)
+        project_import.send(sender=project, request=request)
+        trigger_initial_build(project, request.user)
```

Comment:
I wonder if this should all be abstracted into a single function called `trigger_build(initial)` and used in all the builds, so we only have to change this logic in one place.